docs/logs/index.md

```diff
@@ -1,54 +1,39 @@
 title: Sematext Logs
 description:  Cloud based SaaS / On-premises logging as a service platform. Index machine data, search, correlate and investigate. Build in reports and dashboards with monitoring and alerting capabilities.
 
-Think of Sematext Logs as logging as a service or your own logging box in the cloud (or [on your own infrastructure](../sematext-enterprise) if you really can't ship logs out to the cloud).
+Sematext Logs is a **Log Management-as-a-service**. Think of it as your own **central location for logs** in the cloud. If you want to have the same but on your own infrastructure, check out our [Enterprise](../sematext-enterprise) product.
 
-It is cheaper alternative to Splunk, or even as Hosted Elasticsearch, since one of the APIs Logsene exposes is [Elasticsearch API for indexing](index-events-via-elasticsearch-api) and [searching](searching-log-events).
+You can collect logs from any part of your software stack. By using log shippers you centralize and index logs from all parts in one single place, [Sematext Logs](../logs).
```

Comment:
Just an FYI sort of thing, @adnanrahic - logs are not necessarily from one's software stack.  They are also often from network gear, like routers, switches, etc.  Such devices can be configured to ship their logs (typically syslog format) to some host:port, so one can configure them to ship logs to Sematext.

Reply:
Added that info too.